sos/plugins/openvswitch.py

```diff
@@ -42,7 +42,10 @@ def setup(self):
             # Gather the database.
             "ovsdb-client -f list dump",
             # List the contents of runtime directory
-            "ls -laZ /var/run/openvswitch"
+            "ls -laZ /var/run/openvswitch",
+            # Gather systemd services logs
+            "journalctl -u openvswitch",
+            "journalctl -u openvswitch-nonetwork"
```

Comment:
That's the plan but we'd like to centralize it so that individual plugin author's don't need to worry about getting it right & we can apply a policy to all plugins.

I've not tried anything out yet but I'm imagining something like:

```python
    def add_journald_unit(unit, maxlines=None):
      # put code here
```

Used like:

```python
        self.add_journald_unit("openvswitch-nonetwork")
```

Or:

```python
        self.add_journald_unit("openvswitch-nonetwork", lines=1000)
```

For plugins that have a need to override the policy/global line limits.